metvocab/mmdgroup.py

```diff
@@ -0,0 +1,94 @@
+"""
+MetVocab : MMD Group Class
+==========================
+
+Copyright 2021 MET Norway
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from metvocab.cache import DataCache
+
+
+class MMDGroup():
+
+    def __init__(self, voc_id, uri):
+
+        self._voc_id = voc_id
+        self._uri = uri
+
+        self._cache = DataCache()
+        self._concepts = {}
+
+        self._data = self._cache.get_vocab(voc_id, uri)
+
+        # Populate _concepts with data-dictionary for members of uri-group
+        for graph in self._data.get("graph", []):
```

Comment:
Didn't we agree to populate data in a separate function?

Reply:
Was originally in function called in __init__. Should we make it a method not called by __init__?

Reply:
I think all classes should behave the same way. Makes it easier to use.

Reply:
Roger